/**:
  ros__parameters:
    # Vehicle Parameters
    vehicle_length: 2.5
    vehicle_width: 1.8
    max_speed: 5.0
    max_omega: 1.5
    
    # Environment Parameters
    x_min: -10.0
    x_max: 10.0
    y_min: -10.0
    y_max: 10.0
    
    # Planning Parameters
    grid_resolution: 0.1
    rrt_max_iter: 5000
    rrt_step_size: 0.5
    rrt_search_radius: 1.0
    
    # PID Controller Parameters
    kp_v: 1.0
    ki_v: 0.1
    kd_v: 0.05
    kp_w: 2.0
    ki_w: 0.2
    kd_w: 0.1
    
    # LQR Controller Parameters
    Q_diag: [1.0, 1.0, 1.0, 0.5, 0.5]
    R_diag: [0.1, 0.1]
    
    # MPC Controller Parameters
    horizon: 10
    dt: 0.1
    mpc_Q_diag: [10.0, 10.0, 5.0, 1.0, 1.0]
    mpc_R_diag: [0.1, 0.1]
    
    # CBF Safety Filter Parameters
    alpha: 0.5
    safety_margin: 0.5
    max_solver_time: 0.01
    
    # Topics (not used as params, hardcoded in nodes)
    # Control Loop Rate
    control_rate: 50.0

